pushjack/apns.py

```diff
@@ -503,7 +535,24 @@ def to_dict(self):
             'thread-id': self.thread_id
         })
 
-        return message
+        return msg
+
+    def to_dict(self):
+        """Return message as dictionary."""
+        if self.message is None or self.max_payload_length == 0:
+            return self._construct_dict()
```

Comment:
Passing of `self.message` is missing from `self._construct_dict()` which is what's causing the test failure on Travis.

I'd probably move the message trimming to its own method though and then modify the logic to something like:

```python
data = self._construct_dict(message)

if self.max_payload_length:
    data = self._truncate_dict(data)

return data
```

Reply:
Oh, good catch, thank you! Initially the condition only contained the first check for self.message, so it wasn't necessary there. It was only later that I added the check for max_payload_length, and forgot to pass self.message to the call. I fixed it.